src/Main.java

```diff
@@ -1,61 +1,144 @@
+        byte bananas = 5;
+        byte weightOfOneBanana = 80;
+        short totalWeightBanana = (short) (bananas * weightOfOneBanana);
+
+        short TotalMilkInLiters = 200;
+        byte oneHundredGramsOfMilk = 100;
+        byte amountOfOneHundredGramsOfMilk = (byte) (TotalMilkInLiters / oneHundredGramsOfMilk);
+        byte oneHundredLitersInGams = 105;
+        short numberOfGramsOfMilk = (short) (amountOfOneHundredGramsOfMilk * oneHundredLitersInGams);
+
+        byte allIceCream = 2;
+        byte weightOfOneIceCream = 100;
+        short totalWeightCream = (short) (weightOfOneIceCream * allIceCream);
+
+        byte totalEggs = 4;
+        byte weightOfOneEgg = 70;
+        short weightOfEggs = (short) (totalEggs + weightOfOneEgg);
+
+        short mixtureGrams = (short) (totalWeightBanana + numberOfGramsOfMilk + totalWeightCream + weightOfEggs);
+        float kilogramСonversion = 0.001f;
+        float mixtureKilogram = (float) (mixtureGrams * kilogramСonversion);
+        System.out.println(mixtureGrams + " грамм");
+        System.out.println(mixtureKilogram + " килограмм");
+
+        System.out.println();
+        System.out.println("Задание 7");
+        byte needToReset = 7;
+        short conversionToGramWeight = 1000;
+        short needToResetGram = (short) (needToReset * conversionToGramWeight);
+        short athleteLittleLose = 250;
+        short athleteLoseLot = 500;
+        short LosesFewDays = (short) (needToResetGram / athleteLittleLose);
+        short LosesManyDays = (short) (needToResetGram / athleteLoseLot);
+        System.out.println("Eсли спортсмен будет терять каждый день по 250 грамм " + LosesFewDays
+                + " дней уйдет на похудение.");
+        System.out.println("Eсли спортсмен будет терять каждый день по 500 грамм " + LosesManyDays
+                + " дней уйдет на похудение.");
+        byte averageDays = (byte) ((LosesFewDays + LosesManyDays) / 2);
+        System.out.println("В среднем может потребоваться " + averageDays
+                + " дней, чтобы добиться результата похудения!");
+
+        System.out.println();
+        System.out.println("Задание 8");
+        int MashaGets = 67760;
+        int DenisGets = 83690;
+        int ChristinaGets = 76230;
+
+        float tenPercent = 1.1f;
+        float MashaIncreased = (MashaGets * (float) tenPercent);
+        float DenisIncreased = (DenisGets * (float) tenPercent);
+        float ChristinaIncreased = (ChristinaGets * (float) tenPercent);
+
+
+        byte billingPeriod = 12;
+        int MashaAnnualCurrent = (MashaGets * billingPeriod);
+        int DenisAnnualCurrent = (DenisGets * billingPeriod);
+        int ChristinaAnnualCurrent = (ChristinaGets * billingPeriod);
+
+
+        float MashaAnnualIncreased = (float) (MashaIncreased * billingPeriod);
+        float DenisAnnualIncreased = (float) (DenisIncreased * billingPeriod);
+        float ChristinaAnnualIncreased = (float) (ChristinaIncreased * billingPeriod);
```

Comment:
```suggestion
        float mashaAnnualIncreased = (float) (MashaIncreased * billingPeriod);
        float denisAnnualIncreased = (float) (DenisIncreased * billingPeriod);
        float christinaAnnualIncreased = (float) (ChristinaIncreased * billingPeriod);
```
названия переменных надо писать с маленькой буквы